.golangci.yml

```diff
@@ -22,8 +22,5 @@ linters:
     - varcheck
 
 linters-settings:
-  gocyclo:
```

Comment:
As noted in the PR description, it's because:

> as the search experiments API has gotten complicated

The latest changes to the search experiments API add another conditional. One way to get around it would be to move some of the logic into helper functions. But also, the choice of 25 earlier was arbitrary. So, for now I've relaxed the check to use the default value of 30.